static/app/views/organizationStats/teamInsights/issues.tsx

```diff
@@ -91,8 +95,11 @@ function TeamStatsIssues({location, router}: Props) {
 
             <DescriptionCard
               title={t('New and Returning Issues')}
-              description={t(
-                'The new, regressed, and unignored issues that were assigned to your team.'
+              description={tct(
+                'The new, regressed, and [status] issues that were assigned to your team.',
+                {
+                  status: hasEscalatingIssues ? 'unarchived' : 'unignored',
```

Comment:
it might make more sense for `unarchived` to be `escalating` to keep the terminology consistent